cmd/main.go

```diff
@@ -212,28 +221,106 @@ func run(cmd *cobra.Command, args []string) error {
 	wgConsumer.Add(1)
 	go func() {
 		defer wgConsumer.Done()
-		printer(ch)
+		for res := range ch {
+			zombies = append(zombies, res)
+		}
 	}()
 
 	wgProducer.Wait()
 	close(ch)
 	wgConsumer.Wait()
 
-	return nil
+	return zombies, nil
```

Comment:
I think dumping results while discovering has its advantages, especially on big clusters. While on the other hand this change enables us to pretty print a nice table.
I would prefer a new argument called `--no-stream`. If given it would cache results like done in this pr. By default results would be dumped immediately.